### PR TITLE
[Bug-7950] Add unit test for AlertPluginInstanceController

### PR DESCRIPTION
**Pull request by springmonster**
## Purpose of the pull request

Fix issue https://github.com/apache/dolphinscheduler/issues/7950

## Brief change log

Add Unit Test for AlertPluginInstanceController

## Verify this pull request

This pull request is already covered by existing tests, such as AlertPluginInstanceControllerTest

This change added tests and can be verified as follows:

Add Unit Test for AlertPluginInstanceController

### CODE REVIEW

**Comment by springmonster** [comment]
image

File: dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/controller/AlertPluginInstanceControllerTest.java [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.content;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+/**
+ * alert plugin instance controller test
+ */
+public class AlertPluginInstanceControllerTest extends AbstractControllerTest {
+    private static final Gson gson = new Gson();
```
Comment (zhuangchong) [inline dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/controller/AlertPluginInstanceControllerTest.java]: It is recommended to use the system tool class JSONUtils.

Comment (springmonster) [reply dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/controller/AlertPluginInstanceControllerTest.java]: Dear @zhuangchong , modified by following your suggestion.
Please check again, thanks very much!

**Review by zhuangchong — APPROVED** [review]
+1

**Pull request merged**